### ISSUE TITLE
Missing 1 required positional argument: 'api_key'

### ISSUE THREAD

**Issue by euclip** [pypingou/fedora-active-user#18]
This error seems to happen on line 107:

> ERROR:active-user:__init__() missing 1 required positional argument: 'api_key'
> Traceback (most recent call last):
>   File "/home/mkjp/git/fedora-active-user/fedora_active_user.py", line 494, in main
>     _get_bugzilla_history(args.email)
>   File "/home/mkjp/git/fedora-active-user/fedora_active_user.py", line 107, in _get_bugzilla_history
>     bzclient = Bugzilla(url='https://bugzilla.redhat.com/xmlrpc.cgi')
> TypeError: __init__() missing 1 required positional argument: 'api_key'

There is a single step to reproduce: try to search for an active user with their email address.

**Comment by pypingou** [comment]
Which version of python-bugzilla do you have?

**Comment by euclip** [comment]
Since the dependency of python-bugzilla is not mentioned in the readme or any other obvious place, I installed the packaged the script seems to use, which was just called Bugzilla. That's how I got this error. I now uninstalled that, and installed python bugzilla 3.0.2 with which I now get a different error:

> ERROR:active-user:cannot import name 'Bugzilla' from 'bugzilla' (unknown location)

**Comment by euclip** [comment]
I decided to try this again after a reboot, and the error's gone now. Couldn't tell you why, but this seems to have been something on my end.

**Comment by pypingou** [comment]
Thanks for the follow-up. Those bugs are always weird and annoying.

I'll note to update the README for the dependency on python-bugzilla though,
that sounds like something that should be there.

**Comment by euclip** [comment]
Yeah, well... I'm not so sure about that. I went back and checked, and the `fedora-packager` package does pull in `python-bugzilla` as an indirect dependency. For whatever reason it wasn't working for me at first after installing the package, and I tried to mess with packages, and even with pip, to make the import work. Long story short, all I had to do was reboot my computer. I also checked with a fresh VM just now, and just installing the fedora-packager package is normally enough.
